recipe/meta.yaml

```diff
@@ -23,27 +23,34 @@ requirements:
     - setuptools
     - wheel
     - cython
-    - fftw
     - {{ pin_compatible('numpy') }}
+    - fftw 3.3.9
   run:
+    - python
     - _openmp_mutex  # [linux]
     - llvm-openmp    # [osx]
```

Comment:
I don't think these two are needed anymore.

Reply:
setup.py does look for openmp
https://github.com/pavlin-policar/openTSNE/blob/v1.0.1/setup.py#L164

Reply:
Oops! Thanks Charles!